monai/networks/blocks/segresnet_block.py

```diff
@@ -9,28 +9,31 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from typing import Union
+from typing import Tuple, Union
 
 import torch.nn as nn
 
 from monai.networks.blocks.convolutions import Convolution
 from monai.networks.blocks.upsample import UpSample
-from monai.networks.layers.factories import Act, Norm
-from monai.utils import InterpolateMode, UpsampleMode
+from monai.networks.layers.factories import Act, Norm, split_args
+from monai.utils import InterpolateMode, UpsampleMode, has_option
 
 
-def get_norm_layer(spatial_dims: int, in_channels: int, norm_name: str, num_groups: int = 8):
-    if norm_name not in ["batch", "instance", "group"]:
-        raise ValueError(f"Unsupported normalization mode: {norm_name}")
-    if norm_name == "group":
-        norm = Norm[norm_name](num_groups=num_groups, num_channels=in_channels)
-    else:
-        norm = Norm[norm_name, spatial_dims](in_channels)
-    if norm.bias is not None:
-        nn.init.zeros_(norm.bias)
-    if norm.weight is not None:
-        nn.init.ones_(norm.weight)
-    return norm
+def get_norm_layer(spatial_dims: int, in_channels: int, norm: Union[Tuple, str]):
```

Comment:
Can we make it to be a `utils function` and share it between networks? I see several networks have this duplicated logic now.